lib/master_manipulator/version.rb

```diff
@@ -1,5 +1,5 @@
 module MasterManipulator
   module Version
-    STRING = '1.2.6'
+    STRING = '1.2.7'
```

Comment:
This isn't' necessary. The MM release process will bump the version number (in a rake task).